nvpair/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "nvpair"
-version = "0.5.0"
+version = "0.5.1"
```

Comment:
I wonder if there's a way to indicate that this is an unreleased version, maybe something like `"0.5.1delphix0`?  That way nothing will get confused if upstream releases an 0.5.1 that's different from what we're doing here.  (This probably applies to the work we've already done, as well.)